sormas-base/pom.xml

```diff
@@ -802,6 +802,12 @@
 				<scope>test</scope>
 			</dependency>
 
+			<dependency>
+				<groupId>com.github.tomakehurst</groupId>
+				<artifactId>wiremock</artifactId>
+				<version>1.58</version>
```

Comment:
Seems that the version you used here is from 2015. Any reason for this? There are to be some vulnerabilities for versions before 2.16.0 https://www.cvedetails.com/cve/CVE-2018-9116/